example/package.json

```diff
@@ -26,7 +26,7 @@
     "@react-navigation/stack": "^6.0.7",
     "@segment/sovran-react-native": "^0.2.0",
     "react": "17.0.2",
```

Comment:
Does React need to be updated too?

Reply:
no, that's what the diff website is showing. Still trying to figure out what's going on with the Android test